R/auditClassicalBenfordsLaw.R

```diff
@@ -335,28 +341,34 @@ auditClassicalBenfordsLaw <- function(jaspResults, dataset, options, ...) {
         "benford" = log10(1 + 1 / digits),
         "uniform" = 1 / length(digits)
       )
-      tb[["lb"]] <- rep(".", length(digits))
-      tb[["ub"]] <- rep(".", length(digits))
       tb[["pval"]] <- rep(".", length(digits))
       tb[["bf"]] <- rep(".", length(digits))
+      if (options[["confidenceInterval"]]) {
+        tb[["lb"]] <- rep(".", length(digits))
+        tb[["ub"]] <- rep(".", length(digits))
+      }
```

Comment:
You probably left the existing structure intact for now, but just FYI, `tb$setExpectedSize(length(digits))` would have done this for all columns.

Reply:
But it does not give the dots (`.`) in the table, right?

<img width="569" alt="image" src="https://github.com/jasp-stats/jaspAudit/assets/25059399/5ec67116-1bae-43ae-a72b-967ec83d4fa1">

Reply:
Nope, it does not. Perhaps we should allow people to specify a placeholder symbol, @JorisGoosen what do you think? So `tb$setExpectedSize(length(digits), placeholder = ".")`.